O21.Game/U95/Level.fs

```diff
@@ -0,0 +1,24 @@
+﻿module O21.Game.U95.Reader
+
+open System
+open System.Collections.Generic
+open System.Threading.Tasks
+open Microsoft.Xna.Framework.Graphics
+open O21.Game.U95.Parser
+
+type Level = {
+    LevelMap: List<List<Option<Texture2D>>>
+}
```

Comment:
I'd say that it's better to make some data structure or enum for map items, and expose the level as a 2D array of them, and not textures.